### DISCUSSION TITLE
烏克蘭無人機之戰：人類戰爭或進入全自動殺手機器人時代

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#1037] [General]
距俄羅斯與烏克蘭之間的俄烏戰爭已經爆發將近一年之久，但雙方仍未達成共識停火，近期兩國更是擴大使用無人機交戰，無人機毫無人性道德可言，當一台無人機自頭頂飛過，一旦判斷到友人的蹤跡，立刻鎖定目標，不到1秒的時間直接擊殺，在未來，可成會演變AI大戰，兩方互拚誰的演算法快，雖然AI戰爭也有一些好處，諸如透過精確定位來減少戰爭中的平民傷亡，但可能會出現的最壞結果，是更難以掌控的人與人、人與機器、機器與機器之間相互交織的危機，可能就真的成了科電影中，人類大戰AI的局面了。

連結:https://technews.tw/2023/01/05/human-warfare-may-enter-the-era-of-fully-autonomous-killer-robots
![image](https://user-images.githubusercontent.com/113499537/211138996-5d1325d6-d170-4b4f-8bb4-4a85ad70daf9.png)

**Comment by dabbie109303501** [comment]
會有這樣罔顧人命的AI機器人都是使用者的錯
眼看著能造福人類的科技因為人的貪慾成為了戰爭兵器真是讓人痛心
糟蹋了研發人員的用心良苦
那個引發戰爭的俄羅斯領導人為何就不能消停一下呢

**Comment by Valentinaa0214** [comment]
因為無人機可以在戰爭中繼續飛行而不需要飛行員，使戰爭變得更加冷酷。
無人機空襲後的殘骸或彈坑都有可能對當地民眾造成更長期危害。
原本用於造福人類社會的科技發明被用於戰爭用途，是相當不可取的，希望國際能正視這個問題。
